roles/robottelo/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I think this should be defaults not vars if the desire is for them to be configurable? Or at least, the ones that we can envision a user wanting to such as test_arguments

Reply:
Good find. I started with copies from downstream, didn't notice it was vars instead of defaults.